### PR TITLE
Add 112.txt

### PR DESCRIPTION
**Pull request by mikhailmits**

### CODE REVIEW

**Comment by MihailSL** [comment]
I need one more file: add 113.txt and leave 112.txt

File: 112.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+112.txt
```
Comment (MihailSL) [inline 112.txt]: Change this file looks formatting like 111.txt

**Review by mikhailmits — COMMENTED** [review]
Nice

**Review by MihailSL — CHANGES_REQUESTED** [review]
Do it

**Comment by mikhailmits** [comment]
Did it!

**Comment by MihailSL** [comment]
Very nice! I'm merging.

**Pull request merged**